Lib/tkinter/ttk.py

```diff
@@ -1543,11 +1543,12 @@ def destroy(self):
         try:
             self._variable.trace_vdelete('w', self.__tracecb)
         except AttributeError:
-            # widget has been destroyed already
             pass
         else:
             del self._variable
-            Frame.destroy(self)
+        super().destroy()
```

Comment:
Parent's `destroy()` now is called even if this `destroy()` already was called. I.e. it can be called twice.